package.json

```
@@ -92,6 +92,18 @@
                     "scope": "resource",
                     "type": "array"
                 },
+                "black-formatter.showNotification": {
+                    "default": "on-warn",
```

```suggestion
                    "default": "on-error",
```
I believe this should be `on-error`, and when we do show notification there should be a way to add a button to the displayed message to trigger a setting change. This might be a limitation of the LSP, but if we can run a command for VS Code we could make this work. That is probably a separate thing.

/cc @luabud this is a prime candidate for experiment.